.github/actions/poetry_deps/action.yaml

```diff
@@ -21,6 +21,7 @@ runs:
     - name: Install Poetry dependencies
       working-directory: ${{ inputs.poetry-dir }}
       run: |
+        poetry env use python3.9
```

Comment:
i think we should add a comment here on why we need this here.

Reply:
incorporated